Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16-alpine as development
+FROM node:14-alpine as development
```

Comment:
Why the drop down to 14 here?

Reply:
14 is the new 12 - or another way of being confused at times :D